### PR TITLE
List 0.5.0 breaking changes already available in earlier releases.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -11,15 +11,43 @@ Breaking Changes:
...
-
+Breaking changes already present in earlier releases as "experimental 0.5.0" (sorted by importance):
+ * General: Disallow legacy constructor syntax (use ``constructor`` keyword).
+ * General: Require ``emit`` prefix when calling events.
+ * General: Scoping rules change from function-based (JavaScript) to block-based (C99/C++-style).
+ * Type Checker: Contract types do not include members from ``address`` anymore.
+ * Type Checker: Make ``keccak256``, ``sha256`` and ``ripemd160`` require a single bytes argument (use ``abi.encodePacked(...)``).
+ * Type Checker: Enforce ``view`` and ``pure``.
+ * Code Generator: Use ``STATICCALL`` opcode for calling ``view`` and ``pure`` functions.
+ * Code Generator: Assert that ``k != 0`` for ``mulmod(a, b, k)`` and ``addmod(a, b, k)`.
```
Comment (erak) [inline Changelog.md]: There's a backtick missing at the end of this line.

File: Changelog.md [hunk 2]
```diff
@@ -11,15 +11,43 @@ Breaking Changes:
 Language Features:
  * General: Allow appending ``calldata`` keyword to types, to explicitly specify data location for arguments of external functions.
 
-Bugfixes:
-
-
+Breaking changes already present in earlier releases as "experimental 0.5.0" (sorted by importance):
```
Comment (axic) [inline Changelog.md]: I think this should read something like `via the "experimental 0.5.0" mode, but now they are not optional anymore`.

**Comment by chriseth** [comment]
Updated.

File: Changelog.md [hunk 3]
```diff
@@ -25,8 +25,40 @@ Language Features:
 Compiler Features:
  * Type Checker: Show named argument in case of error.
 
+Breaking changes already present in earlier releases via the "experimental 0.5.0" mode, which are not optional anymore (sorted by importance):
```
Comment (axic) [inline Changelog.md]: Why not place this after the "breaking changes" block and have it sorted alphabetically (as the rest)?

Comment (chriseth) [reply Changelog.md]: I think I sorted it by impact for the developer.

**Pull request closed without merging**